packages/main/src/plugin/input-quickpick/input-quickpick-registry.ts

```diff
@@ -86,7 +86,7 @@ export class InputQuickPickRegistry {
       deferred.reject('Input has been cancelled');
     });
     // return the promise
-    return deferred.promise;
+    return deferred.promise as Promise<string | undefined>;
```

Comment:
there is no check to assume this cast

Reply:
Fixed by dd0c481bb95d4d5e8c7c2925b3b44e63480290e8